go-controller/pkg/ovn/address_set/address_set.go

```diff
@@ -131,11 +131,12 @@ func (asf *ovnAddressSetFactory) DestroyAddressSetInBackingStore(name string) er
 	if err != nil {
 		return err
 	}
-	err = destroyAddressSet(getIPv4ASName(name))
+	ip4ASName, ip6ASName := MakeAddressSetName(name)
+	err = destroyAddressSet(ip4ASName)
 	if err != nil {
 		return err
 	}
-	err = destroyAddressSet(getIPv6ASName(name))
+	err = destroyAddressSet(ip6ASName)
```

Comment:
I think we should try to delete as much as possible and then return the error, this way if ipv4 fails we never try to delete ipv6
```
	var errstrings []string
	ip4ASName, ip6ASName := MakeAddressSetName(name)
	err = destroyAddressSet(ip4ASName)
	if err != nil {
		errstrings = append(errstrings, err.Error())
	}
	err = destroyAddressSet(ip6ASName)
	if err != nil {
	errstrings = append(errstrings, err.Error())
	}
	if len(errstrings) > 0 {
		return fmt.Errorf(strings.Join(errstrings, "\n"))
	}
	return nil
```

Reply:
done